src/App.js

```diff
@@ -1,13 +1,29 @@
 import "./App.css";
 import Header from "./components/Header";
-import Body from "./components/Body";
 import Footer from "./components/Footer";
+import Reviews from "./components/Reviews";
+import Review from "./components/Review";
+import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
 
 function App() {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
suggest you use separate isLoading states for each component, as otherwise it'll be re-rendering every component with the `isLoading` state; it only needs to load with each separate api call.

Reply:
(this should also get rid of those pesky warnings in the compiler)